jerry-core/include/jerryscript-debugger.h

```diff
@@ -33,7 +33,8 @@ extern "C"
 typedef enum
 {
   JERRY_DEBUGGER_SOURCE_RECEIVE_FAILED = 0, /**< source is not received */
-  JERRY_DEBUGGER_SOURCE_RECEIVED = 1, /**< the source has been received */
+  JERRY_DEBUGGER_SOURCE_RECEIVED = 1, /**< a source has been received */
+  JERRY_DEBUGGER_NO_MORE_SOURCE = 2, /**< no more source signal received */
```

Comment:
We can not use the plural form here, because that is declared as a client message. Perhaps we should choose an other name for this one. Like JERRY_DEBUGGER_SOURCE_NONE ?